### PR TITLE
Add example for animation-delay CSS property

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds an example for [`animation-delay`](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-delay). Let me know if you want to see any changes. Thanks!

### CODE REVIEW

File: live-examples/css-examples/animation/animation-delay.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#example-element {
+    background-color: #1766aa;
+    color: white;
+    margin: auto;
+    margin-left: 0;
+    border: 5px solid #333;
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+
```
Comment (wbamberg) [inline live-examples/css-examples/animation/animation-delay.css]: I'm not sure why there's a blank line here.

File: live-examples/css-examples/animation/meta.json [hunk 2]
```diff
@@ -9,7 +9,18 @@
             "title": "CSS Demo: animation",
             "type": "css"
         },
-        "animation-fill-mode": {
+        "animationDelay": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/animation/animation-delay.css",
+            "jsExampleSrc":
+                "../../live-examples/css-examples/animation/animation-fill-mode.js",
```
Comment (wbamberg) [inline live-examples/css-examples/animation/meta.json]: Maybe it's worth giving this JS file a generic name?

File: live-examples/css-examples/animation/meta.json [hunk 3]
```diff
@@ -9,7 +9,18 @@
...
+            "exampleCode": "live-examples/css-examples/animation/animation-delay.html",
+            "fileName": "animation-delay.html",
+            "title": "CSS Demo: animation",
```
Comment (wbamberg) [inline live-examples/css-examples/animation/meta.json]: This should be "CSS Demo: animation-delay"

File: live-examples/css-examples/animation/animation-delay.css (hunk 1 above)
Comment (ddbeck) [reply live-examples/css-examples/animation/animation-delay.css]: Me neither! I'll remove it.

**Comment by ddbeck** [comment]
OK, I've fixed the issues mentioned. I wasn't sure if there was a "good" name for the shared JS file, so if you've got a better name in mind, let me know. Thanks!

**Pull request merged**